data/campaigns/Eastern_Invasion/scenarios/10_Dark_Sanctuary.cfg

```diff
@@ -0,0 +1,2776 @@
+            [/note]
+        [/objectives]
+
+        [remove_shroud]
+            side=1
+            x=30-38, 39-44,44,   45,   46,   47,   48,   49,   50,   51,   52,   53,   54-99
+            y=20-99, 16-99,19-99,18-99,18-99,19-99,18-99,18-99,17-99,18-99,17-99,17-99,16-99
+        [/remove_shroud]
+        [lift_fog]
+            side=1
+            x=30-38,39-58
+            y=20-32,16-32
+            multiturn=yes
+        [/lift_fog]
+    [/event]
+
+    [event]
+        name=start
+
+        [message]
+            speaker=Dacyn
+            message= _ "We have arrived. This is a long-abandoned sanctum that you may know as the lair of the dark mage Iliah-Malal."
+        [/message]
+        [message]
+            speaker=Dacyn
+            # once again Dacyn is an unreliable narrator and does not speak absolute truth, he only says what he knows
+            # in fact the Amulet by itself can't do what he says it does, that's just his assumption
+            message= _ "It is here that I seek Iliah-Malal’s Amulet of the Veil, an artifact possessing the power to open gates between the land of the living and that of the dead.
+
+I once thought such a power too dangerous to use; I am no longer so naive."
+        [/message]
+        [message]
+            speaker=Owaec
+            message= _ "Iliah-Malal? I know that name from the legends..."
+        [/message]
+
+        [delay]
+            time=2000
+        [/delay]
+
+        [message]
+            speaker=Owaec
+            message= _ "All this time... all this time I thought we were questing towards a noble goal. Searching for a legendary sword, an ancient tome, a holy relic."
+        [/message]
+        [message]
+            speaker=Owaec
+            message= _ "Instead, our quest brings us to the lair of one of the most evil beings that ever lived. How can you possibly mean to make use of anything ever possessed by such an evil? As soldiers of Wesnoth we are honor-bound to fight for the light, not the darkness!"
+        [/message]
+        [message]
+            speaker=Dacyn
+            message= _ "Do not be so shortsighted. With the power to open portals to the Land of the Dead, I can pull Mal-Ravanal’s spirit out of this world. Then, the spells granting false life to the dead will fail, and their fearsome armies will collapse into piles of bones. I am wielding this power for the good of Irdya, not for your honor’s sake."
+        [/message]
+        [message]
+            speaker=Gweddry
+            message= _ "Dacyn, I am uneasy about this course of action. I know little of the Land of the Dead, but everything I have heard suggests-"
+        [/message]
+        [message]
+            speaker=Owaec
+            message= _ "You would violate the deads’ eternal peace to fight a war, yet you call me shortsighted?! And while our homeland bleeds, you have the gall to speak of the good of all life?! Madness! As a mage of the Light, you should know better!"
```

Comment:
Dead can be plural, so I think it should be 'violate the dead's eternal peace'

Reply:
Rather than plural, "dead" here would be a mass noun. Though it leads to the same conclusion.